spec/controllers/hearings_controller_spec.rb

```diff
@@ -498,7 +498,7 @@
   end
 
   describe "#show" do
-    let!(:hearing) { create(:hearing, :with_tasks, scheduled_time: "8:30AM") }
+    let!(:hearing) { create(:hearing, :with_tasks, scheduled_time: "8:30 AM Pacific Time(US & Canada)") }
```

Comment:
For this test, I think it can be changed to something like this nwo that the columns have been added:


```suggestion
    let!(:hearing) { create(:hearing, :with_tasks, scheduled_time: "8:30 AM", scheduled_in_timezone: "Pacific Time (US & Canada)") }
```

Reply:
I'll update the test in this format. As a note- I think going forward the tests should all be in a similar format. For now the tests are still failing since the scheduled_for change is separate from this pull request